### ISSUE TITLE
IMPORT coco annotations with relative file path

### ISSUE THREAD

**Issue by uwer** [jsbroks/coco-annotator#456]
- I have imported several thousand annotations but none of the images could be found as they have a relative path directive.
Furthermore - annotations residing on the server can not be loaded as the only way is to upload annotation files which seems counter intuitive as the images are supposed be not on the local host.
- I also very much second #400 - including relative file path!
- feature request - add importing of annotated archives including relative annotation files in subfolders

**Comment by Alloooshe** [comment]
did you manage to solve this ?

**Comment by uwer** [comment]
Nope, actually left this for a while - just got back to it. I started to rewrite the filename directive in the image map, and it is correct as far as the container is concerned - but it still is not fining the files (after i copied the over and they are registered).
I am still looking for some documentation as to what the rules are for import - without trawling code ...

**Comment by uwer** [comment]
@Alloooshe  - i have added relative pathnames for import and export here https://github.com/uwer/coco-annotator
